src/components/Navigations/MenuButton.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+import MaterialIcon, { type Icon } from '@/components/MaterialIcon.vue'
+
+defineProps<{
+  icon: Icon,
+  text: string
+}>()
+const emit = defineEmits(['click'])
+
+function onClick() {
+  emit('click')
+}
+</script>
+
+<template>
+  <button
+    class="flex w-full items-center gap-2 px-4 py-1"
```

Comment:
ボタンの角丸と、ホバー時の背景を設定しておいてほしいです。

```suggestion
    class="flex w-full items-center gap-2 rounded-lg px-4 py-1 hover:bg-brand-light-primary"
```